connector/src/main/java/org/geysermc/connector/entity/ItemEntity.java

```diff
@@ -39,12 +39,30 @@ public ItemEntity(long entityId, long geyserId, EntityType entityType, Vector3f
         super(entityId, geyserId, entityType, position, motion, rotation);
     }
 
+    @Override
+    public void spawnEntity(GeyserSession session) {
```

Comment:
Is this so the position has its offset applied correctly? If so, you could just apply `this.position = Vector3f.from(position.getX(), position.getY() + 0.3f, position.getZ())` to the end of the class constructor and that should do the same thing.

Reply:
I don't believe this applies if the `setPosition` suggestion I mentioned in my main comment works.

Reply:
An offset of 0.3f was added because Bedrock's ItemEntity bounces sideways in Bedrock, unlike Java, when thrown at the corners as if the collider was a bit larger.

Reply:
This comment was focusing on I don't think you need to override spawnEntity in order to do that - just add the line I proposed to the bottom of the class constructor.

Reply:
Checking again, moveRelative doesn't seem to work. I will modify this part.